packages/checkout-com/src/useCkoCard.ts

```diff
@@ -14,7 +14,15 @@ const setCardToken = (token) => localStorage.setItem(getCardTokenKey(), token);
 const removeCardToken = () => localStorage.removeItem(getCardTokenKey());
 
 const useCkoCard = () => {
-  const makePayment = async ({ cartId }) => {
+  const fetchAvailableMethods = async (reference) => {
```

Comment:
I'm not sure if I get this part right.
Is the return value of this call locale-independent? I can't find the context creation part in cko docs, thus the question.
Either way, wouldn't it be better to create this context when `useCkoCard` is called and returnavailable methods as readonly prop? 

By the way - considering this one bit is added - is this composable still card-related only or should it be renamed from `useCkoCard` to `useCko`?

Reply:
We need billing address on the server to create a context. So I think autosending create context request might be wrong way for some devs.

About naming, yes, maybe we should separate them to 2 composables `useCkoCard` and `useCko` - for fetchAvailableMethods.

Tell us what do you think @andrzejewsky

Reply:
One more suggestion:

From DX perspective it would be good to get here api similar to `useProduct` for example.
Meaning `useCko` would return readonly `availableMethods` and a method `loadMethods` that would populate the former.
This way dev wouldn't have to care about assigning the result of fetching available methods.
What do you think?

Reply:
first thing - we shouldn't load payment methods inside of composable that means payment method  itself 😃 @Fifciu let's discuss about it on the call